docs/jwst/mrs_imatch/LICENSE.rst

```diff
@@ -0,0 +1,5 @@
+*******
+LICENSE
+*******
+
+.. include:: ../../../LICENSE
```

Comment:
No, we haven't decided on a convention for licensing files for JWST docs. I'm open to suggestions or examples from how things are done for, say, the hstcal package.